packages/fetchai/agents/aries_faber/aea-config.yaml

```diff
@@ -8,13 +8,13 @@ fingerprint: {}
 fingerprint_ignore_patterns: []
 connections:
 - fetchai/http_client:0.6.0
-- fetchai/oef:0.7.0
+- fetchai/p2p_libp2p:0.6.0
+- fetchai/soef:0.6.0
 - fetchai/stub:0.7.0
 - fetchai/webhook:0.5.0
 contracts: []
 protocols:
 - fetchai/default:0.4.0
-- fetchai/fipa:0.5.0
 - fetchai/http:0.4.0
 - fetchai/oef_search:0.4.0
 skills:
```

Comment:
I don't think the default connection is correct here

Reply:
Faber uses default message to send Alice some information.